### ISSUE TITLE
[Bug] Blurriness/fuzziness/pixelation during video playback

### ISSUE THREAD

**Issue by Cee123** [YTVanced/Vanced#221]
**Bug description**
I'm not sure what's happening with this, but on my cousin's Amlogic device, there's a lot of blurriness during YouTube video playback and it just happened all of a sudden. It went from being crystal clear to being blurry and fuzzy. I've tried changing the video codec to v9 and the quality settings to override maximum quality but it didn't do anything.

**Variant**
NonRoot: Yes
Root: No

**Vanced version**
15.43.32 - Build - 01.47.00

**Android version**
Android 5.1.1

**Device**
Amlogic s905 quad-core cortex-A53 2.0 Ghz
Mali-450 5-Core GPU
2GB DDR3 16GB EMMC
2.4G +5G WIFI,802.11 B/G/N
100/1000M LAN
Bluetooth4.0
Hardward 3D graphics acceleration
2K*4K Support
Dolby/ 3G/ SPDIF/ IEC958/ Coaxial

**Steps to Reproduce**
Not sure, it just happened to occur suddenly without really changing anything.

**Expected behavior**
It was clear previously, but some unknown reason, it's not now.

_Attach images/videos if possible. These can be helpful in solving the issue._
Unfortunately I don't have any photos as it keeps occurring on my cousin's device. I'll obtain some photos and upload them here. I'd take some photos of mine, but I have exactly the same device but on mine playback is clear.

**Comment by gghhkm** [comment]
Try changing ExoPlayer v2 to v1

**Comment by Cee123** [comment]
> 
> 
> Try changing ExoPlayer v2 to v1

I'll try that, and see if it makes a difference. Thank you. But I think I might've tried that already.

**Comment by Cee123** [comment]
Well I tried that and it didn't help. Still blurry. I'll try clearing some space and try reinstalling the manager and the and the YouTube Vanced again. Although it's baffling as to why it seems to be doing that.

**Comment by Cee123** [comment]
I resolved the issue by
1. Uninstalled vanced manager, vanced microG and YouTube vanced
2. Restarted the device
3. Installed the old vanced manager, version 1.4.1
4. Through the vanced manager, i installed vanced microG and YouTube vanced again
5. In the settings i changed the codec to the V9
6. Video settings i override the maximum quality, set both wifi and other connection quality to 720p and the video play speed as normal

So far all seems to be working well.

**Comment by Cee123** [comment]
Blurriness is STILL a problem with YouTube Vanced. I've tried changing every possible setting under settings, but to no avail. No matter what I change, it's still fuzzy and blurry. About the only thing that seems to work is a full uninstallation and reinstallation and sometimes that doesn't even work and I have to try it several times.

**Comment by gghhkm** [comment]
> Blurriness is STILL a problem with YouTube Vanced. I've tried changing every possible setting under settings, but to no avail. No matter what I change, it's still fuzzy and blurry. About the only thing that seems to work is a full uninstallation and reinstallation and sometimes that doesn't even work and I have to try it several times.

A new build is going to come out in the near future, so fingers crossed it'll fix it for you.

**Comment by Cee123** [comment]
> > Blurriness is STILL a problem with YouTube Vanced. I've tried changing every possible setting under settings, but to no avail. No matter what I change, it's still fuzzy and blurry. About the only thing that seems to work is a full uninstallation and reinstallation and sometimes that doesn't even work and I have to try it several times.
> 
> A new build is going to come out in the near future, so fingers crossed it'll fix it for you.

Nice! Hopefully there will be a fix.

**Comment by Cee123** [comment]
Playback is still blurry and pixelated... I tried everything. Tried changing the codec, overriding max quality settings, overriding the make and model, uninstalling, reinstalling, wiping the cache and the data and reloading YouTube Vanced, tried changing the Exoplayer version. I can't even change the quality on the videos as the option isn't there or available. I tried everything and nothing's working... videos are just staying pixelated and blurry.
